tasks/main.yml

```diff
@@ -59,12 +59,13 @@
   setup:
   when: "{{ ansible_docker0 is not defined or docker_config_result | changed }}"
 
+- name: login to docker registry
```

Comment:
Don't do this coz this is what we use for creating AMIs and they would have the info that is the specific reason why we did not add login stuff into this role.